# Case C→C
An Associated Press article captured the attention of readers with the headline "Night lights bad for kids?" The article was based on a 1999 study at the University of Pennsylvania and Children's Hospital of Philadelphia, in which parents were surveyed about the lighting conditions under which their children slept between birth and age 2 (lamp, night light, or no light) and whether or not their children developed nearsightedness (myopia). The purpose of the study was to explore the effect of a young child's nighttime exposure to light on later nearsightedness.
<br>
In this activity, we use the collected data to:
<br>
 - Learn how to build a two-way table and compute conditional percentages.
 - Interpret the data in terms of the relationship between a young child's nighttime exposure to light and later nearsightedness.

In [1]:
%matplotlib inline

In [2]:
import pandas as pd

In [39]:
data = pd.read_excel('https://oli.cmu.edu/repository/webcontent/8aaa78340a0001dc5574e39e401d2529/_u2_summarizing_data/webcontent/excel/nightlight.xls')

In [40]:
data.head()

Light Nearsightedness
0     no light              No
1  night light              No
2         lamp              No
3     no light              No
4     no light              No

**Explanatory variable is Light (categorical) and Response Variable is Nearsigtedness (categorical)**

In [41]:
light_near = pd.crosstab(index=data['Light'],columns=data['Nearsightedness'],margins=True)
light_near.columns = ['No','Yes','rowtotal']
light_near.index = ['lamp','night light','no light','coltotal']
light_near


No  Yes  rowtotal
lamp          34   41        75
night light  153   79       232
no light     155   17       172
coltotal     342  137       479

In [42]:
light_near['No'] = light_near['No']/light_near['rowtotal']
light_near['Yes'] = light_near['Yes']/light_near['rowtotal']

In [43]:
light_near

No       Yes  rowtotal
lamp         0.453333  0.546667        75
night light  0.659483  0.340517       232
no light     0.901163  0.098837       172
coltotal     0.713987  0.286013       479

In [44]:
# In percentage
light_near[['No','Yes']] * 100

No        Yes
lamp         45.333333  54.666667
night light  65.948276  34.051724
no light     90.116279   9.883721
coltotal     71.398747  28.601253

Of the three groups of children, those who were exposed to no light at all were least likely to be nearsighted, with an incidence of 9.9%. Those exposed to a night-light were nearsighted 34.1% of the time, and those exposed to a lamp were nearsighted 54.7% of the time. Note that these percentages show that children who slept with a lamp were about 5 times more likely to develop nearsightedness than the children who slept with no light.
<br>

Based upon this data alone, parents might discontinue using night-lights and lamps with young children, as their use seems to be associated with nearsightedness when the children grow up. However, there is a strong argument against this conclusion, which will be presented in a later lesson.